webextensions/manifest/theme.json

```diff
@@ -866,18 +866,14 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/theme#images",
             "support": {
               "chrome": {
-                "version_added": true
-              },
-              "edge": {
-                "version_added": "79"
+                "version_added": "≤68"
```

Comment:
@queengooborg this looks as if it was added in  #2194 which is dated Jul 12, 2018 so according to the version history I have would be 67 as version 68 was introduced on 2018-07-24

Reply:
For these PRs, I've been going off of the date when the PR had _landed_ rather than when it was created, just to be on the safe side -- but I'd say that in cases like this where no changes were made between the PR creation date and its merge, we can go based on the PR creation date!

Reply:
Understood